pkg/processor/sumologicschemaprocessor/README.md

```diff
@@ -32,6 +32,17 @@ processors:
     # See `translate_telegraf_metrics_processor.go` for full list of translations.
     # default = true
     translate_telegraf_attributes: {true, false}
+
+    # Specifies if attributes should be nested, basing on their keys.
+    # See "Nesting attributes" documentation chapter from this document.
+    nest_attributes:
+      # Defines whether attributes should be nested.
+      # default = false
+      enabled: {true, false}
+
+      # Defines the string used to separate key names in attributes that are to be nested.
+      # default = "."
+      separator: <separator>
```

Comment:
In my opinion, we could have some pattern matching here. So in case customer wants to leave some keys which has separator in it he can do i, but current behavior seems to be reasonable for first iteration

Reply:
This has been already commented this on Slack. IMO we should add this in another PR (it is also arguable what should the filters be: we can do an allowlist or a denylist, with advanced matching options or not... imo it's the best to just add a denylist of prefixes)